firefox_ui_harness/runtests.py

```diff
@@ -68,9 +68,6 @@ def startTestRunner(runner_class, options, tests):
         if options.installer:
             installer = os.path.realpath(options.installer)
 
-            if not mozinstall.is_installer(installer):
```

Comment:
Well, I missed that this method is already called inside of `mozinstall.install()`. Sorry for not mentioning it.